src/internal/views/nunjucks/billing/batch-error.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "./nunjucks/layout.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "./nunjucks/billing/macros/batch-header.njk" import batchHeader %}
+
+{% block content %}
+
+  {{ govukErrorSummary({
+    titleText: "There is a problem with this bill run",
+    errorList: errorList
+  }) }}
+
+  {{ batchHeader(pageTitle, batch) }}
+
+  <p>
```

Comment:
Would we normally add a class to this? https://design-system.service.gov.uk/styles/typography/#paragraphs
```suggestion
  <p class="govuk-body">
```